Content.Server/Storage/EntitySystems/DumpableSystem.cs

```diff
@@ -39,6 +45,21 @@ private void OnAfterInteract(EntityUid uid, DumpableComponent component, AfterIn
             if (storage.StoredEntities == null || storage.StoredEntities.Count == 0)
                 return;
 
+            if (TryComp<MaterialStorageComponent>(args.Target, out var materialStorage))
+            {
+                foreach(var mat in materialStorage!.EntityWhitelist!.Tags!)
+                {
+                    foreach (var serv in storage!.Whitelist!.Tags!)
```

Comment:
That's a lot of questionable null suppression, also you shouldn't be matching whitelists like that because it comprises components and tags and this just ignores components entirely.

Reply:
1) Compiler don't allow me to use output vars in another ways. I tried, same "can't convert links"
2) "comprises components and tags and this just ignores components entirely"
I don't get it. 
2.1) Do you mean that this "include" tags for tag list?
2.2) Ignores what components? That many that is inside a bag?

Reply:
huh?